solutions/usaco-923.mdx

```diff
@@ -0,0 +1,420 @@
+---
+id: usaco-923
+source: USACO Gold 2019 February
+title: Painting the Barn
+author: Kevin Sheng
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_paintbarn_gold_feb19.html)
+
+## Explanation
+
+### Initial Observations
+
+**Note:** This solution assumes one has already had a firm grasp of the
```

Comment:
corny, maybe reword it ngl

Reply:
corny?

Reply:
reword it so you the average 3rd grader can understand it easier

Reply:
cope